src/CAREUI/display/Timeline.tsx

```diff
@@ -52,6 +52,10 @@ interface TimelineNodeProps {
 
 export const TimelineNode = (props: TimelineNodeProps) => {
   const name = useContext(TimelineContext);
+  let newName: string | null = null;
+  if (props.name === "daily round details Event") {
+    newName = props.name.replace("daily round", "log update");
+  }
```

Comment:
This is not the right way to solve the problem. 

It should instead be changed in the props passed to this component.

Reusable components such as these should not contain any implementation specific to a particular usage.